packages/truffle-artifactor/README.md

```diff
@@ -43,30 +31,31 @@ $ npm install truffle-artifactor
 
 ### Example
 
-Here, we'll generate a `.sol.js` files given a JSON object like [truffle-schema](https://github.com/trufflesuite/truffle-schema). This will give us a file which we can later `require` into other projects and contexts.
+Here, we'll generate a `.json` files given a JSON object like [truffle-contract-schema](https://github.com/trufflesuite/truffle/tree/develop/packages/truffle-contract-schema). This will give us a file which we can later `require` into other projects and contexts.
 
 ```javascript
-var artifactor = require("truffle-artifactor");
+const Artifactor = require("truffle-artifactor");
+const artifactor = new Artifactor(__dirname);
 
-// See truffle-schema for more info: https://github.com/trufflesuite/truffle-schema
-var contract_data = {
+// See truffle-schema for more info: https://github.com/trufflesuite/truffle/tree/develop/packages/truffle-contract-schema
+const contract_data = {
   abi: ...,              // Array; required.
   unlinked_binary: "..." // String; optional.
   address: "..."         // String; optional.
 };
 
-artifactor.save(contract_data, "./MyContract.sol.js").then(function() {
-  // The file ./MyContract.sol.js now exists, which you can
-  // import into your project like any other Javascript file.
-});
+artifactor.save(contract_data);
+// The file ./MyContract.json now exists, which you can
+// import into your project like any other Javascript file.
 ```
+
 # API
 
-#### `artifactor.save(options, filename[, extra_options])`
+#### `artifactor.save(contract_data)`
```

Comment:
Might as well start using camelCase here

Reply:
Oh btw! We should update the code examples to use current schema. It's now `contractName`, and the like.